Sources/TuistLoader/Models+ManifestMappers/RunActionOptions+ManifestMapper.swift

```diff
@@ -13,11 +13,15 @@ extension TuistGraph.RunActionOptions {
         generatorPaths: GeneratorPaths
     ) throws -> TuistGraph.RunActionOptions {
         var language: String?
+        var region: String?
+        var showNonLocalizedStringsAsUppercase: Bool
         var storeKitConfigurationPath: AbsolutePath?
         var simulatedLocation: SimulatedLocation?
         var enableGPUFrameCaptureMode: GPUFrameCaptureMode
 
         language = manifest.language?.identifier
+        region = manifest.region
+        showNonLocalizedStringsAsUppercase = manifest.showNonLocalizedStringsAsUppercase
```

Comment:
Note that this is not sufficient, you still need to adjust [this logic](https://github.com/tuist/tuist/blob/78367e26028506407c393c657580ba30386fa0d4/Sources/TuistGenerator/Generator/SchemeDescriptorsGenerator.swift#L396) to set the right value in the generated scheme.

Reply:
you mean when XcodeProj has been updated? If so, yes I know. If not, then I'm not sure I understand :)

Reply:
There's a bit of indirection that might explain the confusion. The following mappings happen in the process of generating a project.
- `RunActionOptions (ProjectDescription)` to `RunActionOptions (TuistGraph)`: This maps the public models from ProjectDescriptiont to the ones that Tuist uses internally, which are slightly different from the public ones. For example, the internal ones have the glob patterns resolved.
- `RunActionOptions (TuistGraph)` to `XCScheme.LaunchAction`: This maps the internal model to the actual representation in the generated Xcode project.

The **later is missing**. You'll have to update XcodeProj to the latest version, which I have just released, and then update [the logic here](https://github.com/tuist/tuist/blob/78367e26028506407c393c657580ba30386fa0d4/Sources/TuistGenerator/Generator/SchemeDescriptorsGenerator.swift#L603) to include the new attribute. Let me know if it's clearer now.

Reply:
working on it now :)